src/image/p5.Image.js

```diff
@@ -681,92 +877,228 @@ p5.Image = class {
   }
 
   /**
-   * Applies an image filter to a <a href="#/p5.Image">p5.Image</a>
+   * Applies an image filter to the <a href="#/p5.Image">p5.Image</a> object.
+   * The preset options are:
    *
-   * THRESHOLD
-   * Converts the image to black and white pixels depending if they are above or
-   * below the threshold defined by the level parameter. The parameter must be
-   * between 0.0 (black) and 1.0 (white). If no level is specified, 0.5 is used.
+   * `INVERT`
+   * Inverts the colors in the image. No parameter is used.
    *
-   * GRAY
-   * Converts any colors in the image to grayscale equivalents. No parameter
-   * is used.
+   * `GRAY`
+   * Converts the image to grayscale. No parameter is used.
    *
-   * OPAQUE
-   * Sets the alpha channel to entirely opaque. No parameter is used.
+   * `THRESHOLD`
+   * Converts the image to black and white. Pixels with a grayscale value
+   * above a given threshold are converted to white. The rest are converted to
+   * black. The threshold must be between 0.0 (black) and 1.0 (white). If no
+   * value is specified, 0.5 is used.
    *
-   * INVERT
-   * Sets each pixel to its inverse value. No parameter is used.
+   * `OPAQUE`
+   * Sets the alpha channel to entirely opaque. No parameter is used.
    *
-   * POSTERIZE
-   * Limits each channel of the image to the number of colors specified as the
-   * parameter. The parameter can be set to values between 2 and 255, but
-   * results are most noticeable in the lower ranges.
+   * `POSTERIZE`
+   * Limits the number of colors in the image. Each color channel is limited to
+   * the number of colors specified. Values between 2 and 255 are valid, but
+   * results are most noticeable with lower values. The default value is 4.
    *
-   * BLUR
-   * Executes a Gaussian blur with the level parameter specifying the extent
-   * of the blurring. If no parameter is used, the blur is equivalent to
-   * Gaussian blur of radius 1. Larger values increase the blur.
+   * `BLUR`
+   * Blurs the image. The level of blurring is specified by a blur radius. Larger
+   * values increase the blur. The default value is 4. A gaussian blur is used
+   * in `P2D` mode. A box blur is used in `WEBGL` mode.
    *
-   * ERODE
+   * `ERODE`
    * Reduces the light areas. No parameter is used.
    *
-   * DILATE
+   * `DILATE`
    * Increases the light areas. No parameter is used.
    *
-   * filter() does not work in WEBGL mode.
-   * A similar effect can be achieved in WEBGL mode using custom
-   * shaders. Adam Ferriss has written
-   * a <a href="https://github.com/aferriss/p5jsShaderExamples"
-   * target='_blank'>selection of shader examples</a> that contains many
-   * of the effects present in the filter examples.
+   * `img.filter()` uses WebGL in the background by default because it's faster.
+   * This can be disabled in `P2D` mode by adding a `false` argument, as in
+   * `img.filter(BLUR, false)`. This may be useful to keep computation off the GPU
```

Comment:
I think we didn't actually add support for WebGL hardware acceleration of `p5.Image` yet, so we can probably take out this paragraph for now

Reply:
Oh oops, good catch!